tests/test_physical_channel_property.py

```diff
@@ -0,0 +1,65 @@
+"Contains a collection of pytest tests that validates the physical channel properties."
+import os
+
+from nidaqmx.constants import TerminalConfiguration
+from nidaqmx.tests.test_read_write import TestDAQmxIOBase
+
+
+class TestPhysicalChannelProperty(TestDAQmxIOBase):
+    """Contains a collection of pytest tests that validates the physical channel properties."""
```

Comment:
Remove the test class and use test functions. I don't think this test is using any of the base class's methods.